integrade/tests/api/v1/urls.py

```diff
@@ -5,3 +5,4 @@
 AUTH_TOKEN_CREATE = '/auth/token/create/'
 AUTH_TOKEN_DESTROY = '/auth/token/destroy/'
 AUTH_USERS_CREATE = '/auth/users/create/'
+API_CLOUD_ACCOUNT = '/api/v1/account/'
```

Comment:
Let's do some brainstorm here. Since this is the first time an endpoint from api is added here and all endpoints but auth related are going to start with `API_` what if we drop that and name based on the API path? So `/api/v1/account/` would be `ACCOUNT`, `/api/v1/something/else/` would be `SOMETHING_ELSE`?